vendor/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,13 +20,35 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
~AFAICT this change is fine, but might be worth noting / keeping an eye on~

This may be a problem https://github.com/bazelbuild/bazel-gazelle/blob/44ce230b3399a5d4472198740358fcd825b0c3c9/language/go/fileinfo.go#L388